src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block-stars stars--0">
+      <div class="block-stars__stars"></div>
```

Comment:
For the future.
There is no necessary to add `block` prefix for block names. 
You can just name: `stars`, `stars__star`.